benchmarks/media-streaming/server/Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM cloudsuite/base-os:debian
 
 RUN apt-get update -y \
-  && apt-get install -y --no-install-recommends nginx \
+  && apt-get install -y --no-install-recommends nginx openssl \
   && rm -rf /var/lib/apt/lists/*
 
 # Increase the open file limit
 COPY files/limits.conf.append /tmp/
-RUN cat /tmp/limits.conf.append >> /etc/security/limits.conf
+RUN cat /tmp/limits.conf.append >> /etc/security/limits.confi
```

Comment:
Why is this changed? Is it a typo?

Reply:
Yes. Sorry. I pushed another commit to fix it.